Website/htdocs/mpmanager/database/seeders/UserSeeder.php

```diff
@@ -15,10 +15,18 @@ class UserSeeder extends Seeder
      */
     public function run()
     {
+
+        $randomPass = str_random(8);
         DB::table('users')->insert([
             'name' => 'Admin',
             'email' => 'admin@admin.com',
-            'password' => Hash::make('basic-password'),
+            'password' => Hash::make($randomPass),
         ]);
+
+        $this->command->alert('Your Random Password is generated.');
+        $this->command->info('Your credential is: ');
```

Comment:
Can you replace it with a single line output like?


Please use following credentials to login:
Email
Password


And please add an additional command 
login:reset-password that resets the password for the case that the user doesn't store the password.